composer.json

```diff
@@ -63,6 +63,7 @@
             "Phpactor\\Extension\\LanguageServerCompletion\\Tests\\": "tests/LanguageServerCompletion/",
             "Phpactor\\Extension\\LanguageServerReferenceFinder\\Tests\\": "tests/LanguageServerReferenceFinder/",
             "Phpactor\\Extension\\LanguageServerIndexer\\Tests\\": "tests/LanguageServerIndexer/",
+            "Phpactor\\Extension\\LanguageServerRename\\Tests\\": "tests/LanguageServerRename/",
```

Comment:
The dev autoloading is a bit strange in this repo, but having this helps Phpactor with various class creation tasks.